Dangerfile

```diff
@@ -58,11 +58,12 @@ end
 
 modified_files.each do |file|
   begin
+    ignoreWarn = !!(file =~ /.*\/(drawable|layout)\/.*/)
     fileDiff = git.diff_for_file(file)
     # Ensure we keep using secure https:// references instead of http://
     httpMatches = fileDiff.patch.scan(/\+.*http:\/\/.*/)
     httpMatches.each do |httpMatch|
-      warn("Detected unsecure `http://` use in `#{file}` section `#{httpMatch}`") if httpMatch
+      warn("Detected unsecure `http://` use in `#{file}` section `#{httpMatch}`") if httpMatch && !ignoreWarn
```

Comment:
@melloflavio 
I am doing this transition to functions in another branch #30 
Is okay if I replicate these changes in the other branch after rebasing, since I already created the function?